packages/ui-extensions/buildTargetDts.ts

```diff
@@ -159,22 +163,44 @@ function updateReferences(
   node.replaceWithText(text);
 }
 
+function getTargets(sourceFile: SourceFile) {
+  const base = sourceFile.getInterface('ExtensionTargets')!;
+  const structure = base.getStructure();
+  const allInterfaces = [base];
+
+  if (Array.isArray(structure?.extends)) {
+    structure.extends.forEach((inherited) => {
+      if (typeof inherited === 'string') {
+        allInterfaces.push(sourceFile.getInterface(inherited)!);
+      }
+    });
+  }
+
+  return allInterfaces;
+}
+
```

Comment:
Same fix applied in this [customer-accounts PR](https://github.com/Shopify/ui-extensions/pull/2759). Thank you @brianshen1990 🙏.